.pre-commit-config.yaml

```diff
@@ -25,6 +25,11 @@ repos:
        exclude: ^litellm/tests/|^litellm/proxy/tests/
        additional_dependencies: [flake8-print]
        files: litellm/.*\.py
+    # -  id: flake8
```

Comment:
this should be on ci/cd, not just a pre-commit hook @ishaan-jaff

Reply:
looks like a simple ruff check solves this 

just add this to your ruff.toml 

```
select = ["PLR0915"]  
```

Reply:
<img width="1382" alt="Screenshot 2024-10-12 at 12 54 04 PM" src="https://github.com/user-attachments/assets/0e20363f-e19f-4fcb-9d5c-9630aa231774">